forc/src/utils/defaults.rs

```diff
@@ -100,12 +100,7 @@ abigen!(MyContract, "out/debug/"#,
 
 async fn get_contract_instance() -> (MyContract, ContractId) {
     // Launch a local network and deploy the contract
-    let mut wallets = launch_custom_provider_and_get_wallets(
-        WalletsConfig::new_single(Some(1), Some(1000000)),
-        None,
-    )
-    .await;
-    let wallet = wallets.pop().unwrap();
+    let wallet = launch_provider_and_get_wallet().await;
```

Comment:
Do we not need to pass in coins anymore? @digorithm recommended a few weeks ago to do that which is why this was changed to specify a custom provider.

Reply:
I see. There are 2 different ways to do this from what I understand. I believe this is the simpler of the 2, but has UTXO validation disable IIRC.
I can switch to using the other method if that's what was recently recommended.